src/components/account/AccountSetting.jsx

```diff
@@ -26,11 +81,17 @@ const AccountSetting = () => {
                 type="text"
                 placeholder="Your First Name"
                 className="input input-bordered input-primary border-black w-full max-w-xs text-black placeholder:text-black/70"
+                value={userData.first_name || ""}
+                onChange={handleInputChange}
+                name="first_name"
               />
               <input
                 type="text"
                 placeholder="Your Last Name"
                 className="input input-bordered input-primary border-black w-full max-w-xs text-black placeholder:text-black/70"
+                value={userData.last_name || ""}
```

Comment:
For future refactor -
This seems redundant since you're already setting the default state to be "" ?
Definitely something to come back to when you have the time